test/custom_output_prefix.t/run.t

```diff
@@ -0,0 +1,2183 @@
+  $ dune exec -- stork gen -o custom_output_prefix original_output_prefix_1.atd original_output_prefix_bar.atd
```

Comment:
I love this!! Although maybe let's split it into separate files? What do you think? This is quite overwhelming.